src/main/java/com/exactpro/epfast/template/xml/package-info.java

```diff
@@ -0,0 +1,20 @@
+/******************************************************************************
+ * Copyright 2019-2020 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+@XmlSchema(namespace = NamespaceProvider.XML_NAMESPACE)
```

Comment:
I think `namespace` attribute can now be removed from all `@XmlElement` annotations.

Reply:
I tried, it didn't work